src/daemon/control.rs

```diff
@@ -508,7 +508,7 @@ pub fn presigned_tx_sighash(
     hashtype: SigHashType,
 ) -> Result<secp256k1::Message, SigError> {
     // Presigned transactions only have one input when handled by revaultd.
-    if !tx.tx().input.len() == 1 {
+    if tx.tx().input.len() != 1 {
```

Comment:
Wtf wasn't this part of master already?

Reply:
You modified it again in d23681d58ecb70774ae52fa9637372fbd4a559f2 I think. Or maybe I had troubles with rebasing...

Reply:
Should probably be squashed then, it's confusing to have both